go/vt/vtgate/planbuilder/symboltable.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2016, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package planbuilder
+
+import (
+	"errors"
+
+	"github.com/youtube/vitess/go/vt/sqlparser"
+)
+
+// SymbolTable contains the symbols for a SELECT
```

Comment:
For now, it's table names. But I don't know what else we'll store. I think we'll also store SELECT alias names. So, staying non-commital for now.